vendor/bat-native-ads/src/bat/ads/internal/ad_targeting/data_types/behavioral/bandits/epsilon_greedy_bandit_arms.cc

```diff
@@ -63,6 +63,11 @@ EpsilonGreedyBanditArmMap GetArmsFromDictionary(
   }
 
   for (const auto value : dictionary->DictItems()) {
+    if (value.first.empty()) {
+      NOTREACHED();
```

Comment:
@moritzhaller, maybe this is the reason of crashes. 
Maybe we should changes all NOTREACHED here to BLOG, because all this cases could be reached in case of wrong preferences.  WDYT?